### ISSUE TITLE
Providing `nodeExecCommand` setting

### ISSUE THREAD

**Issue by ptrdom** [scalacenter/scalajs-bundler#427]
This is somewhat related to https://github.com/scalacenter/scalajs-bundler/issues/418 - instead of directly supporting some node version manager, we could expose a setting that would allow users to set a `node` executable, similar to how `sbt-native-packager` does it with [dockerExecCommand](https://github.com/sbt/sbt-native-packager/blob/c87195443cd3f5942ac7889bd2048f70efda3a0d/src/main/scala/com/typesafe/sbt/packager/docker/Keys.scala#L39). Then users of `scalajs-bundler` could wire in for example https://github.com/jasongin/nvs by setting `nodeExecCommand` to `nvs exec <ver>`.

How does this sound @sjrd?

**Comment by sjrd** [comment]
This is something that can be done with a custom `NodeJSEnv.Config`:
```scala
import org.scalajs.jsenv.nodejs.NodeJSEnv

jsEnv := {
  new NodeJSEnv(NodeJSEnv.Config()
    .withExecutable("nvs")
    .withArgs(List("exec", ver)))
}
```

**Comment by ptrdom** [comment]
When suggesting this I mainly had interactions with Webpack in mind:
- https://github.com/scalacenter/scalajs-bundler/blob/d083e6df3fe2ac795ae16ba69696039bf85f85a3/sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala#L285
- https://github.com/scalacenter/scalajs-bundler/blob/d083e6df3fe2ac795ae16ba69696039bf85f85a3/sbt-scalajs-bundler/src/main/scala/scalajsbundler/WebpackDevServer.scala#L54
It is not possible to intercept node executable used for them.

**Comment by sjrd** [comment]
Wouldn't that be possible to do in user space using #420, then?

**Comment by ptrdom** [comment]
No, because that PR does not expose these bundler related `node` invocations that I have linked above. New setting is the only way. Do not see a problem with here - it is a beneficial change that is easily backwards compatible.
